app/javascript/styles/about.scss

```diff
@@ -120,18 +120,55 @@
 }
 
 .information-board {
-  margin: 20px 0;
-  display: flex;
-  justify-content: space-between;
-  border-top: 1px solid lighten($ui-base-color, 10%);
-  border-bottom: 1px solid lighten($ui-base-color, 10%);
-  padding-right: 14px;
+  background: darken($ui-base-color, 4%);
+  padding: 40px 0;
+
+  .panel {
```

Comment:
Not sure how much you want to fix here, I'm not asking for you to rewrite the whole about.scss file. but moving some of the smaller comments to it would be nice.